<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configWindow</class>
 <widget class="QWidget" name="configWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>285</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/config.png</normaloff>../icons/config.png</iconset>
  </property>
  <widget class="QLabel" name="HostLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Host:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PortLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="UserLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="HostEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>17</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>57</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>97</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>137</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DownloadButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </widget>
  <widget class="QLabel" name="OutputLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
